openrtb_ext/bidders.go

```diff
@@ -115,116 +128,130 @@ const (
-	"adtelligent":       BidderAdtelligent,
-	"advangelists":      BidderAdvangelists,
-	"aja":               BidderAJA,
-	"applogy":           BidderApplogy,
-	"appnexus":          BidderAppnexus,
-	"adoppler":          BidderAdoppler,
-	"avocet":            BidderAvocet,
-	"beachfront":        BidderBeachfront,
-	"beintoo":           BidderBeintoo,
-	"between":           BidderBetween,
-	"brightroll":        BidderBrightroll,
-	"colossus":          BidderColossus,
-	"connectad":         BidderConnectAd,
-	"consumable":        BidderConsumable,
-	"conversant":        BidderConversant,
-	"cpmstar":           BidderCpmstar,
-	"datablocks":        BidderDatablocks,
-	"dmx":               BidderDmx,
-	"emx_digital":       BidderEmxDigital,
-	"engagebdr":         BidderEngageBDR,
-	"eplanning":         BidderEPlanning,
-	"audienceNetwork":   BidderFacebook,
-	"gamma":             BidderGamma,
-	"gamoshi":           BidderGamoshi,
-	"grid":              BidderGrid,
-	"gumgum":            BidderGumGum,
-	"improvedigital":    BidderImprovedigital,
-	"inmobi":            BidderInMobi,
-	"invibes":           BidderInvibes,
-	"ix":                BidderIx,
-	"kidoz":             BidderKidoz,
-	"krushmedia":        BidderKrushmedia,
-	"kubient":           BidderKubient,
-	"lifestreet":        BidderLifestreet,
-	"lockerdome":        BidderLockerDome,
-	"logicad":           BidderLogicad,
-	"lunamedia":         BidderLunaMedia,
-	"marsmedia":         BidderMarsmedia,
-	"mgid":              BidderMgid,
-	"mobilefuse":        BidderMobileFuse,
-	"nanointeractive":   BidderNanoInteractive,
-	"ninthdecimal":      BidderNinthDecimal,
-	"openx":             BidderOpenx,
-	"orbidder":          BidderOrbidder,
-	"pubmatic":          BidderPubmatic,
-	"pubnative":         BidderPubnative,
-	"pulsepoint":        BidderPulsepoint,
-	"rhythmone":         BidderRhythmone,
-	"rtbhouse":          BidderRTBHouse,
-	"rubicon":           BidderRubicon,
-	"sharethrough":      BidderSharethrough,
-	"smaato":            BidderSmaato,
-	"smartadserver":     BidderSmartadserver,
-	"smartrtb":          BidderSmartRTB,
-	"smartyads":         BidderSmartyAds,
-	"somoaudience":      BidderSomoaudience,
-	"sonobi":            BidderSonobi,
-	"sovrn":             BidderSovrn,
-	"synacormedia":      BidderSynacormedia,
-	"tappx":             BidderTappx,
-	"telaria":           BidderTelaria,
-	"triplelift":        BidderTriplelift,
-	"triplelift_native": BidderTripleliftNative,
-	"ucfunnel":          BidderUcfunnel,
-	"unruly":            BidderUnruly,
-	"valueimpression":   BidderValueImpression,
-	"verizonmedia":      BidderVerizonMedia,
-	"visx":              BidderVisx,
-	"vrtcal":            BidderVrtcal,
-	"yeahmobi":          BidderYeahmobi,
-	"yieldlab":          BidderYieldlab,
-	"yieldmo":           BidderYieldmo,
-	"yieldone":          BidderYieldone,
-	"zeroclickfraud":    BidderZeroClickFraud,
+// BidderNames returns all core bidders.
+func BidderNames() []BidderName {
```

Comment:
The new source of truth for bidder names. Matches the model used in the metrics package.